gcloud/datastore/api.py

```diff
@@ -191,15 +193,22 @@ def get(keys, missing=None, deferred=None, connection=None, dataset_id=None):
                        If not passed, inferred from the environment.
 
     :rtype: list of :class:`gcloud.datastore.entity.Entity`
-    :returns: The requested entities.
+    :returns: The requested entities (or a single entity if a single key is
+              passed).
     :raises: EnvironmentError if ``connection`` or ``dataset_id`` not passed,
              and cannot be inferred from the environment.  ValueError if
              one or more of ``keys`` has a dataset ID which does not match
              the passed / inferred dataset ID.
     """
-    if not keys:
+    if not key_or_keys:
         return []
 
+    single_key = not isinstance(key_or_keys, list)
```

Comment:
What about a tuple? or an iterator? Would this be better if we check `single_key = isinstance(key_or_keys, datastore.Key)` ?